content/queries/what-is-a-query.md

```diff
@@ -1,12 +1,12 @@
 # What is a Query?
 
 When we make a request to a GraphQL server to declare our specific data requirements, we are sending a *query*.
-Queries are a central part of Relay. They are consist of fields which are either *scalar* or *non-scalar*.
+Queries are a central part of Relay. They consist of fields which are either *scalar* or *non-scalar*.
 Scalar fields are primitive data types like `String` or `Int`.
 Non-scalar fields consist of non-scalar fields or scalar fields themselves.
 
 *Models* from your GraphQL schema are exposed as non-scalar fields in the query.
-Queries in GraphQL are *hierarchical*, that means that if you include a non-scalar field in a query you have to define a subselection of fields on it.
+Queries in GraphQL are *hierarchical*, that means that if you include a non-scalar field in a query you have to define a subselection of scalar fields on it.
```

Comment:
see 9ad3d3b466a853499beb46359467d3cadb462b96 - I made it slightly clearer and am pretty happy with it now :)